AWS/sagemaker/pytorch_utils/datasets/kannada_dataset.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
+from PIL import Image
+
+import torch
+from torch.utils.data import Dataset
+from torchvision import datasets, transforms
+
+IMGSIZE = 28
+
+
+__all__ = ['KannadaDataset', ]
+
+
+class KannadaDataset(Dataset):
```

Comment:
make sure to go through this during the workshop

Reply:
Yup understood. noted.